Makefile

```
@@ -63,12 +63,14 @@ fmt:
 	@find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
 	@find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/celestiaorg
 	@go mod tidy -compat=1.17
+	@markdownlint --fix --quiet --config .markdownlint.yaml .
 .PHONY: fmt
 
 ## lint: Linting *.go files using golangci-lint. Look for .golangci.yml for the list of linters.
 lint:
 	@echo "--> Running linter"
 	@golangci-lint run
+	@markdownlint --config .markdownlint.yaml .
```

Should these be

```suggestion
	@markdownlint --config .markdownlint.yaml '**/*.md'
```

(ref: https://github.com/igorshubovych/markdownlint-cli/tree/8652eb2bd9f7339f0569cee254fc695b952c7c70#globbing)

---

I've tested it with a single dot multiple times, and it works for all the subdirectories, but I can change this to match specification